database/migrations/2023_12_21_085427_create_view_port_mac_link.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::statement('DROP VIEW IF EXISTS view_port_mac_links;');
```

Comment:
Not sure how I feel about the addition of views.  I know not all databases support them, so if it is a goal to eventually support other databases, they will need to be removed then.

Reply:
Most databases that I'm aware of support simple views like this.  It does pass the MySQL and SQLite tests (I had to make changes to get it to work on the latter).

From memory this was needed to get a significant speed increase in doing the MAC+port lookup.  It has been a long time since I did this, so I'd need to re-discover the original issue to see if I can find another workaround.